code/datums/traits.dm

```diff
@@ -740,6 +740,20 @@ obj/trait/pilot
 	points = 1
 	isPositive = 0
 
+/obj/trait/color_shift
+	name = "Color Shift (0)"
```

Comment:
Not completely sure if I like the trait but I don't have any good arguments for that. I'd like to hear what others think.

Reply:
Fair.

Reply:
I'd say try it out as a trait, if people dislike that it's a trait or something it can be moved to the Spacebux menu or something.

Reply:
Knowing that races are now traits, we can discard the mild complaints from here I reckon.